config/AMBASSADORS_MEMBERS.json

```diff
@@ -1,164 +1,176 @@
-			},
-			{
-				"type": "presentation",
-				"title": "Event-driven APIs & Schema governance for Apache Kafka, Apidays LIVE Hong Kong",
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=rwlHJkgaRdw"
-			},
-			{
-				"type": "presentation",
-				"title": "Event-driven APIs & Schema governance for Apache Kafka, API Specification Conference",
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=QZ4cJyTw-U0"
-			},
-			{
-				"type": "presentation",
-				"title": "Getting started with Event-Driven APIs, Apidays LIVE Paris",
-				"date": {
-					"year": 2021,
-					"month": "December"
-				},
-				"link": "https://www.youtube.com/watch?v=_kIE-qtpwTA&t=9s"
-			}
-		]
-	},
-	{
-		"name": "Jesse Menning",
-		"github": "jessemenning",
-		"twitter": "JesseMenning",
-		"linkedin": "jesse-menning",
-		"company": "Solace",
-		"contributions": [
-			{
-				"type": "article",
-				"title": "AsyncAPI 2.3 Adds Solace Bindings to Unlock Full Power of PubSub+",
-				"date": {
-					"year": 2022,
-					"month": "January"
-				},
-				"link": "https://solace.com/blog/asyncapi-adds-solace-bindings/"
-			},
-			{
-				"type": "article",
-				"title": "Align Production Reality and Event Documentation with the AsyncAPI Discovery Tool",
-				"date": {
-					"year": 2021,
-					"month": "November"
-				},
-				"link": "https://solace.com/blog/asyncapi-discovery-tool/"
-			},
-			{
-				"type": "article",
-				"title": "AsyncAPI vs. OpenAPI: Answers to Your Burning Questions About the Two Leading API Specs",
-				"date": {
-					"year": 2021,
-					"month": "June"
-				},
-				"link": "https://solace.com/blog/asyncapi-vs-openapi/"
-			},
-			{
-				"type": "article",
-				"title": "AsyncAPI, CloudEvents, OpenTelemetry: Which Event-Driven Specs Should Your DevOps Include?",
-				"date": {
-					"year": 2021,
-					"month": "March"
-				},
-				"link": "https://solace.com/blog/asyncapi-cloudevents-opentelemetry-event-driven-specs-devops/"
-			},
-			{
-				"type": "presentation",
-				"title": "Future of AsyncAPI, Panel discussion, AsyncAPI Conference",
-				"date": {
-					"year": 2021,
-					"month": "November"
-				},
-				"link": "https://www.youtube.com/watch?v=3EeMHhbwyOQ"
-			},
-			{
-				"type": "presentation",
-				"title": "Thinking Out Loud #4 — with Jesse Menning",
-				"date": {
-					"year": 2021,
-					"month": "August"
-				},
-				"link": "https://www.youtube.com/watch?v=eJewFUR-OaM"
-			}
-		]
-	}
+  {
+    "name": "Alejandra Quetzalli",
+    "img": "/img/avatars/canela-ale.webp",
+    "bio": "Hecha en 🇲🇽 • 📑OSS DevDocs & Community @asyncAPIspec• 📚Autora @Apress(2023) • Canela🐕‍🦺 es mi #actuallyautistic #ServiceDog • Rebelde🤟🏽• mis opiniones",
+    "title": "DevRel & DevDocs @AsyncAPI",
+    "github": "alequetzalli",
+    "twitter": "QuetzalliAle",
+    "tags": ["presentation", "workshop", "article", "iniciative_driver"],
```

Comment:
what are these tags for?
you have these in the contributions array, so this is unnecessary duplication